### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by 23SRLITheGamer** [raysfire/rayshud#48]
My lighting is very messed up and also its says so much error about not finding stuff. PLS SEND HELP!!!!!!!!

**Comment by electricman226** [comment]
If it has to do with lightly I highly doubt it's rayshud causing the issue, but that's neither here nor there.
I need more than 'PLS SEND HELP', paste in what the console says when it says about errors.

**Comment by 23SRLITheGamer** [comment]
CSoundEmitterSystem:  Registered 7843 sounds
CResponseSystem:  scripts\talker\response_rules.txt (1581 rules, 617 criteria, and 1488 responses)
[TF Workshop] Got 0 subscribed maps, 0 new
CMaterial::PrecacheVars: error loading vmt file materials\Effects\bombinomicon_distortion_dx80.vmt for effects/bombinomicon_distortion
maxplayers set to 24
Heap: 256.00 Mb
Queued Material System: ENABLED!
Steam config directory: C:\Program Files (x86)\Steam\steamapps\common\Team Fortress 2\platform\config
Parsed 26 text messages
CClientSteamContext logged on = 1
missing 'info.vdf'. Removed search path 'c:\program files (x86)\steam\steamapps\common\team fortress 2\tf\custom\my_custom_stuff\' from 'custom_mod' pathID.
missing 'info.vdf'. Removed search path 'c:\program files (x86)\steam\steamapps\common\team fortress 2\tf\custom\trasparent-viewmodels\' from 'custom_mod' pathID.
missing 'info.vdf'. Removed search path 'c:\program files (x86)\steam\steamapps\common\team fortress 2\tf\custom\workshop\' from 'custom_mod' pathID.
KeyValues Error: RecursiveLoadFromBuffer:  got EOF instead of keyname in file resource/ClientScheme.res
Scheme, (*CustomFontFiles*), (*15*), (*name*), (*turkish*), (*range*), (*1*), 
Couldn't find custom font file 'resource/Cerbetica_bold_medic.otf'
Hud element 'CHudItemAttributeTracker' doesn't have an entry 'ItemAttributeTracker' in scripts/HudLayout.res
Hud element 'CTFFlagCalloutPanel' doesn't have an entry 'CTFFlagCalloutPanel' in scripts/HudLayout.res
Hud element 'CTFStreakNotice' doesn't have an entry 'CTFStreakNotice' in scripts/HudLayout.res
Queued Material System: DISABLED!
Queued Material System: ENABLED!
Steamworks Stats: Requesting CLIENT session id.
Requesting twitch.tv streams update...
Cleaning up unneeded replay block data...
Replay cleanup done.
[Cloud]: SUCCEESS retrieved cfg/config.cfg from remote storage into cfg/config.cfg
execing config.cfg
Loading default settings for high sensitivity
[Cloud]: SUCCEESS retrieved materials/vgui/logos/spray.vtf from remote storage into materials/vgui/logos/spray.vtf
[Cloud]: SUCCEESS retrieved materials/vgui/logos/spray.vmt from remote storage into materials/vgui/logos/spray.vmt
Sound Initialization: Start
DS:stereo speaker configuration detected
Set primary sound buffer format: yes
Using secondary sound buffer
   2 channel(s)
   16 bits/sample
   44100 samples/sec
DirectSound initialized
Sound Initialization: Finish, Sampling Rate: 44100
4 CPUs, Frequency: 2.6 Ghz,  Features: GenuineIntel SSE SSE2 MMX RDTSC CMOV FCMOV
Queued Material System: DISABLED!
TF world status heartbeat.
  Event: N
Connection to game coordinator established.
CTFGCClientSystem::PostInitGC
CTFGCClientSystem - adding listener
execing valve.rc
execing autoexec.cfg
execing sixense_bindings.cfg
Hiding LobbyContainerFrameHiding LobbyContainerFrameHiding LobbyContainerFrameChanging resolutions from (1280, 600) -> (1280, 600)
Unable to remove c:\program files (x86)\steam\steamapps\common\team fortress 2\tf\textwindow_temp.html!
Request for twitch.tv streams succeeded (code: 200, size: 9606)...
Steamworks Stats: Received CLIENT session id: 1107019491566
Applying new item schema, version 7E18FBDD
**WARNING** Item schema mismatch after update!
GC told us to expect 7E18FBDD, we got F55AB7D5
Applied updated item schema from GC. 3351812 bytes, version 7E18FBDD.
Request for 'https://static-cdn.jtvnw.net/previews-ttv/live_user_diamond0wner-320x180.jpg' succeeded (code: 200, size: 12669)...
Request for 'https://static-cdn.jtvnw.net/previews-ttv/live_user_bo4r_-320x180.jpg' succeeded (code: 200, size: 13003)...
Request for 'https://static-cdn.jtvnw.net/previews-ttv/live_user_harley_cat-320x180.jpg' succeeded (code: 200, size: 17878)...
Request for 'https://static-cdn.jtvnw.net/previews-ttv/live_user_kdoggery-320x180.jpg' succeeded (code: 200, size: 17224)...
Request for 'https://static-cdn.jtvnw.net/previews-ttv/live_user_reflecto__-320x180.jpg' succeeded (code: 200, size: 16112)...

Here's what shows up when I started up TF2

**Comment by electricman226** [comment]
You installed rayshud incorrectly or haven't updated for the MYM update.
If you recently updated, then you installed incorrectly.
The directory structure should be 'tf/custom/rayshud-master/'.
The readme.txt inside the custom folder is slightly misleading in this case, as the folder contained in the zipped file is what you drag in.

**tl;dr**: Delete my_custom_stuff, redownload rayshud (or if you have it downloaded already that's fine too), and put 'rayshud-master' inside the 'custom' folder; don't put it into it's own directory.

**Comment by 23SRLITheGamer** [comment]
dude the my_custom_stuff is my congaila fps config not my rays hud

**Comment by electricman226** [comment]
Oh.
What does it look like in game?

**Comment by 23SRLITheGamer** [comment]
basically when I'm playing the light is getting distorted so much its annoying and also when I am the sniper the zoom has this ring of distorted color in it

**Comment by electricman226** [comment]
I'm not sure if I'm understanding you clearly. Could you try to get a screenshot in game?

**Comment by 23SRLITheGamer** [comment]
![20160904152344_1](https://cloud.githubusercontent.com/assets/16334980/18229653/1c250f96-72b4-11e6-98af-fac137394f79.jpg)
![20160904152402_1](https://cloud.githubusercontent.com/assets/16334980/18229652/1c1fe48a-72b4-11e6-8354-b9f7d654c9ac.jpg)
![20160904152405_1](https://cloud.githubusercontent.com/assets/16334980/18229659/24578c2a-72b4-11e6-9ec3-167e08945b1a.jpg)
![20160904152407_1](https://cloud.githubusercontent.com/assets/16334980/18229654/1c2a1c66-72b4-11e6-95eb-a02dde99e24d.jpg)
![20160904152414_1](https://cloud.githubusercontent.com/assets/16334980/18229658/20354ab0-72b4-11e6-8b66-10a350564f6e.jpg)

here

**Comment by 23SRLITheGamer** [comment]
see the different lighting

**Comment by electricman226** [comment]
That's rather weird.
I highly doubt that rayshud has anything to do with the problem, it could possibly be your FPS config making the game look different in exchange for performance. The lighting does seem different.

The sniper rifle scope texture overlay color distortion is interesting, though. I'm not so sure what could even be causing that one.

**Comment by 23SRLITheGamer** [comment]
I removed the fps config for this one so the fps config can't do anything here

**Comment by 23SRLITheGamer** [comment]
are you still there

**Comment by electricman226** [comment]
Some convars will save as the config value,  you need to manually reset some convars to there defaults.  There is no way that rayshud is causing this issue; have you even tried the game without it installed?

**Comment by CriticalFlaw** [comment]
This is definitely an issue with your config, a HUD wouldn't change the lighting or anything else in-engine. Try putting -autoconfig in your launch options to reset all tf2 graphical settings to the default.
